README.md

```diff
@@ -105,11 +105,20 @@ gcloud auth login <any-terra-dev-user>
 RUN_AUTHENTICATED_TEST=1 ./scripts/automation-test.sh
 ```
 
-Update/freeze dependencies
+Install Poetry
 ```sh
-scripts/freeze-deps.sh
+brew install poetry
+```
+
+Install dependencies
+```sh
+poetry install
+```
+
+Update dependencies
+```sh
+poetry update
 ```
-This creates a clean virtualenv, installs dependencies from `dependencies-min.txt`, and freezes the resulting environment in `requirements.txt` (which Google App Engine uses during deployment).
```

Comment:
might be useful to replace this comment with one explaining what poetry does here?

Reply:
Sure, I can do that!

Also reading it now... "Google App engine uses during deployment"
Wondering what the implications of my changes are for that...

Reply:
hmmmm yeah [this article](https://cloud.google.com/appengine/docs/standard/python3/specifying-dependencies#:~:text=When%20you%20deploy%20to%20App%20Engine%2C%20the%20dependencies%20specified%20in%20the%20requirements.txt%20file%20will%20be%20installed%20automatically%20with%20your%20deployed%20app.) mentions it. I do think there's some poetry command to export to a txt file, that might be necessary...

Reply:
https://python-poetry.org/docs/cli/#export:~:text=This%20command%20exports%20the%20lock%20file%20to%20other%20formats.